Criterion/Monad.hs

```diff
@@ -19,7 +19,7 @@ module Criterion.Monad
 import Control.Monad.Reader (asks, runReaderT)
 import Control.Monad.Trans (liftIO)
 import Criterion.Monad.Internal (Criterion(..), Crit(..))
-import Criterion.Types hiding (measure)
+import Criterion.Types 
```

Comment:
In general, I'd prefer not to clutter up the diff with minor aesthetic changes like this unless there's a good reason for them (which there doesn't seem to be here).